jupyter_rsession_proxy/__init__.py

```diff
@@ -73,9 +73,10 @@ def db_config(db_dir):
         f.close()
         return db_config_name
 
-    def _support_arg(arg):
+    def _support_args(args):
         ret = subprocess.check_output([get_rstudio_executable('rserver'), '--help'])
-        return ret.decode().find(arg) != -1
+        help_output = ret.decode()
+        return {arg: (help_output.find(arg) != -1) for arg in args}
```

Comment:
```suggestion
        return {arg: (help_output.find(f"--{arg}") != -1) for arg in args}
```

Let's check for `--` instead to slightly increase robustness

Reply:
Yes, more robustness is always good. I will add that.